### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by waltzaround**

### CODE REVIEW

**Comment by MinhPhu0304** [comment]
Friendly version of read me to [read](https://github.com/CovidEngine/vaxxnz/blob/waltzaround-patch-1/CONTRIBUTORS_DEV.md)

**Comment by waltzaround** [comment]
@MinhPhu0304 pls

**Comment by waltzaround** [comment]
will wrap this up after work today

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Once you've joined the Discord, introduce yourself, and let the channel know you would like to contribute in #introductions 🥳
+
+- Have a look at some [open issues](https://github.com/CovidEngine/vaxxnz/issues) and see if anyone needs any help investigating a problem or bug
+
+- [Review our roadmap ](https://github.com/CovidEngine/vaxxnz/projects/2) and join the discussion
+
+- Propose a new idea or feature by [Creating an issue](https://github.com/CovidEngine/vaxxnz/issues)
+
+## For Designers:
```
Comment (oeed) [inline CONTRIBUTORS.md:21]: Maybe also have a link to the developers doc?

File: CONTRIBUTORS_DEV.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Vaxx Nz technical guide
+
+This section of the documentation contains a guide for technicall users who want to contribute code or documentation to the Vaxx.nz. As a community, we share rules of behavior and interaction. Make sure you are familiar with the [community guidelines](CODE_OF_CONDUCT.md) before continuing.
```
Comment (oeed) [inline CONTRIBUTORS_DEV.md:3]: technicall -> technical

File: CONTRIBUTORS_DEV.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## How to get started
+
+### Task 1. Get a GitHub account
+
+To contribute to the Vaxxnz project, you will need a <a
+href="https://github.com" target="_blank">GitHub account</a>. A free account is
+fine. All the Vaxxnz repositories are public and visible to everyone.
+
+You should also have some experience using both the GitHub application and `git`
+on the command line. But if you prefer GUI version which is also fine to.
```
Comment (oeed) [inline CONTRIBUTORS_DEV.md:18]: 'But if you prefer GUI version which is also fine to' is grammatically a bit iffy.

Maybe 'But if you prefer, there's a GUI version available too' or something to that effect.

**Review by oeed — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS_DEV.md (hunk 3 above)
Comment (MinhPhu0304) [reply CONTRIBUTORS_DEV.md:18]: Yeah sounds good, I will make a change to that

File: CONTRIBUTORS.md (hunk 1 above)
Comment (waltzaround) [reply CONTRIBUTORS.md:21]: more for nontechnical work

**Review by oeed — APPROVED** [review]

**Pull request merged**